src/useMemoCache/index.ts

```diff
@@ -14,7 +14,10 @@ type CachedItem<State> = { state: State; dependencyList: DependencyList };
 export const createMemoCache = <State>(
   customAreHookInputsEqual?: typeof nativeAreHookInputsEqual
 ) => {
-  const cache = new Set<CachedItem<State>>();
+  const MAX_ENTRIES = 64;
+  let indexCounter = 0;
```

Comment:
Hello, `indexCounter` is used to determine which element should be removed when threshold is hit

Reply:
Oh, I see. Now I wonder if it would be better to just stop accepting new items to the cache after the maximum size is reached since the cache items are basically equally "important" so there is no reason to be deleting one item instead of another when the cache is filled.